app/utils.js

```diff
@@ -5,7 +5,6 @@ const binary = require('ripple-binary-codec');
 const nodemailer = require('nodemailer');
 const smtpTransport = require('nodemailer-smtp-transport');
 const jsrender = require('jsrender');
-const prova = require('prova-lib');
 const utxoLib = require('bitgo-utxo-lib');
```

Comment:
Do we need to change this to `@bitgo/utxo-lib` and upgrade to a more recent version?

Reply:
Yes, that is done in #90. I think I might just close this PR since the other one removes prova as well as updates the utxo-lib references.